### PR TITLE
Add R tools for windows

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
- Add R language and tools
#### Related issue:
https://github.com/actions/virtual-environments/issues/1220
## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Validate-R.ps1 [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Validate Rscript
+if (Get-Command -Name 'Rscript')
+{
+    Write-Host "Rscript on path"
+}
+else
+{
+    Write-Host 'Rscript is not on path'
+    exit 1
+}
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Validate-R.ps1:15]: Could you please implement new test for this tool?
See https://github.com/actions/virtual-environments/pull/1205

Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Validate-R.ps1:15]: Done.

File: images/win/scripts/Installers/Install-R.ps1 [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+##  File:  Install-R.ps1
+##  Desc:  Install R for Windows
+################################################################################
+
+Choco-Install R.Project
+Choco-Install rtools
+
+$rscriptPath = Resolve-Path "C:\Program Files\R\*\bin\x64"
+Add-MachinePathItem $rscriptPath
+Invoke-PesterTests -TestFile "Tools" -TestName "CMake"
```
Comment (AlenaSviridenko) [inline images/win/scripts/Installers/Install-R.ps1:11]: -TestName should be "R", right?

Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Install-R.ps1]: Done.

File: images/win/scripts/Installers/Validate-R.ps1 [hunk 3]
```diff
@@ -0,0 +1,15 @@
+################################################################################
+##  File:  Validate-R.ps1
+##  Desc:  Validate R.
+################################################################################
+
+# Validate Rscript
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Validate-R.ps1:6]: Should this file be removed?

Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Validate-R.ps1]: Done.

**Pull request merged**